File: Tests/v10/NullableEnabledMixedWithNullableDisabled/NullableEnabledTypes.cs
using KY.Generator;

namespace Types;

[GenerateTypeScriptModel]
public class NullableEnabledTypes
{
    public string? NullableString { get; set; }
    public string NonNullableString { get; set; } = string.Empty;
    public NullableDisabledTypes NullableDisabledTypes { get; set; } = new();
}

File: Tests/NullableEnabledMixedWithNullableDisabled/Output/nullable-enabled-types.ts
/* eslint-disable */
// tslint:disable

import { NullableDisabledTypes } from "./nullable-disabled-types";

export interface NullableEnabledTypes {
    nullableString?: string;
    nonNullableString: string;
    nullableDisabledTypes: NullableDisabledTypes | undefined;
}

// outputid:c5722132-ab5a-4d1f-8673-2b3afb959baf
